cmd/list.go

```diff
@@ -289,3 +310,36 @@ func getOptionalIntField(value int64) string {
 	}
 	return fmt.Sprintf("%d", value)
 }
+
+func sortListData(data [][]any, header []string, sortColumn string) {
+	sortColumn = strings.ToLower(sortColumn)
+	if !slices.Contains(header, sortColumn) {
+		fmt.Println("Warning: column '" + sortColumn + "' does not exist")
+		return
+	}
```

Comment:
That's a great idea, it'll make the code flow much better. I think this should be implemented for #16. The code improvement of #41 could also be done there I think (otherwise the PR would be blocked for some time)